<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="grafico" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>531</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>531</width>
     <height>301</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>531</width>
     <height>301</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="llave">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>83</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>LLAVE -&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>402</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>123</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>MODO -&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modo_grafica">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Segmento de canales</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Canal</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>63</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Maximo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>59</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Minimo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mostrar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maximo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minimo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="helloque">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/hello que.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_cargar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar se√±al</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource/hello_que.qrc"/>
 </resources>
 <connections/>
</ui>
